### PR TITLE
Fixed obsolete API in List Locations section

### PR DESCRIPTION
**Pull request by AlperSakarya**
LocationApi -> LocationsApi

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -46,19 +46,19 @@ from __future__ import print_function
...
 # create an instance of the Location API class
-api_instance = LocationApi()
+api_instance = LocationsApi()
 
 try:
     # ListLocations
     api_response = api_instance.list_locations()
     print (api_response.locations)
 except ApiException as e:
-    print ('Exception when calling LocationApi->list_locations: %s\n' % e)
```
Comment (yonpols) [inline README.md]: Is "LocationApi->list_locations" correct python syntax? It seems like PHP syntax. I know this is an error message but it looks too odd

Comment (AlperSakarya) [inline README.md]: Not quite sure if that's the correct syntax. 

I specifically just changed word "LocationApi" to "LocationsApi", so I just added the letter "s" and made it plural. README.md instructions are working fine after this.

Comment (GrayHatter) [inline README.md]: The syntax in question is existing, and IIRC the same throughout the python api. While it does conflict with the PEP8 recommendations, changing it would be something that should be done in a different pull request.

This change LGTM

**Comment by tristansokol** [comment]
Thanks for your PR! Due to our SDK generation process, I've put these changes in the 2.2.1 release of our specification. They will be propagated to this SDK soon. For future PRs like this please add them to the templates in https://github.com/square/connect-api-specification. 

You can see all the 2.2.1 pending changes here: https://github.com/square/connect-api-specification/pull/65/files
